src/reflect/type.go

```diff
@@ -691,6 +708,23 @@ func (t rawType) Key() Type {
 	panic("unimplemented: (reflect.Type).Key()")
 }
 
+func (t rawType) In(i int) Type {
+	panic("unimplemented: (reflect.Type).In()")
+}
+
+func (t rawType) Out(i int) Type {
+	panic("unimplemented: (reflect.Type).Out()")
+}
+
+func (t rawType) MethodByName(name string) (Method, bool) {
+	panic("unimplemented: (reflect.Type).MethodByName()")
+}
+
+func (t rawType) PkgPath() string {
+	// panic("unimplemented: (reflect.Type).PkgPath()")
+	return "<unimplemented>"
```

Comment:
I need this to return a string for protocol buffers to work, but I'm happy to put the panic back in so the code at least compiles.